jsonschema/_utils.py

```diff
@@ -38,6 +38,22 @@ def __repr__(self):
         return repr(self.store)
 
 
+class Cache(object):
```

Comment:
That should work. This doesn't really need the "lru" portion, but with a sufficiently large value for maxsize, that should be fine.

Apparently both `functools32` and `repoze.lru` have this. I'll look into which ones work on py2.6 

On a related note, do you think I should add a new kwarg to `RefResolver` to disable caching entirely (since there is already an option to disable caching of remote refs) ?